lxd/response.go

```diff
@@ -477,6 +477,7 @@ var NotImplemented = &errorResponse{http.StatusNotImplemented, "not implemented"
 var NotFound = &errorResponse{http.StatusNotFound, "not found"}
 var Forbidden = &errorResponse{http.StatusForbidden, "not authorized"}
 var Conflict = &errorResponse{http.StatusConflict, "already exists"}
+var Unavailable = &errorResponse{http.StatusServiceUnavailable, "unavailable"}
```

Comment:
We shouldn't introduce new response types that don't take error as an argument as we're planning on getting rid of the existing ones.

Can you make that Unavailable(err) where if err is nil, we use "unavailable" as the string?

Reply:
Fixed.